content/docs/configuration/acme/http01/README.md

```diff
@@ -52,18 +52,33 @@ The HTTP01 Issuer supports a number of additional options.  For full details on
 the range of options available, read the [reference
 documentation](../../../reference/api-docs.md#acme.cert-manager.io/v1.ACMEChallengeSolverHTTP01).
 
-### `class`
+### `ingressClassName`
+
+> The field `ingressClassName` was added in cert-manager 1.12.
```

Comment:
Is there an icon we use for these version info boxes?

Reply:
I did not find an existing icon. I added "ℹ️" and an info div.

Note that usually, we use the format:

> **FEATURE STATE:** cert-manager 1.12 [GA]

This format doesn't let the user know when the feature was first introduced (can they use the feature, even if in alpha?).